Assets/Scripts/ConvergeGame/MultiConvergeGame.cs

```diff
@@ -488,38 +562,46 @@ void MakeWindow (int id)
 			}
 		}
 
-		if (GUI.Button (new Rect (bufferBorder + 110, height - 30 - bufferBorder, 110, 30), "Progress Report")) {
+		if (GUI.Button (new Rect (bufferBorder + 90, height - 30 - bufferBorder, 80, 30), "Progress")) {
 			GenerateBarGraph ();
 		}
 
+		if (GUI.Button (new Rect (bufferBorder + 180, height - 30 - bufferBorder, 80, 30), "Winners")) {
+			Debug.Log ("showWinners set to true");
+			showWinners = true;
+		}
+
+
+
+
 		// Display buttons with opponent bet status 
 		Color savedColor2 = GUI.color;
 		float topLeft = topGraph;
 
 		string buttonText;
-		style.fontSize = 14;
+		// style.fontSize = 12;
 		// Debug.Log ("Other player button routine");
+		GUIStyle customButton = new GUIStyle("button");
+		customButton.fontSize = 12;
 		foreach (DictionaryEntry entry in playerNames) {
 			// do something with entry.Value or entry.Key
 			id_otherPlayer = (int) entry.Key;
-			name_otherPlayer = (string)entry.Value;
+			name_otherPlayer = (string)entry.Value + " (won " + roundsWon [id_otherPlayer] + ") ";
 			if ((id_otherPlayer > 0) && (betStatusList.Contains (id_otherPlayer))) {
-
-					if (((short)betStatusList [id_otherPlayer]) == 1) {  // bet placed
-						GUI.color = Color.green; 
-						buttonText = name_otherPlayer + " Made Bet";
-					} else {  // bet not placed
-						GUI.color = Color.red;
-						buttonText = name_otherPlayer + " No Bet";
-					}
-					// Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);
-					// Debug.Log ("Button text: " + buttonText);
-					if (GUI.Button (new Rect (balanceX, topLeft, buttonWidth, 25), buttonText)) {
-						barGraph.setOppName (name_otherPlayer);
-						displayOtherGraph ();
-					}
-					topLeft += buttonStep;
-
+				if (((short)betStatusList [id_otherPlayer]) == 1) {  // bet placed
+					GUI.color = Color.green; 
+					buttonText = name_otherPlayer + " Made Bet";
+				} else {  // bet not placed
+					GUI.color = Color.red;
+					buttonText = name_otherPlayer + " No Bet";
+				}
+				// Debug.Log ("other player button: " + (bufferBorder + width - 170) + " " + topLeft + " " + buttonWidth);
```

Comment:
We can remove the commented out code. Since we are using git for tracking history.